massa-versioning-exports/src/error.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+use displaydoc::Display;
+use massa_models::error::ModelsError;
+use thiserror::Error;
+
+/// versioning error
+#[non_exhaustive]
+#[derive(Display, Error, Debug)]
+pub enum VersioningError {
```

Comment:
VersioningError is too general here maybe rename it to VersioningMiddlewareError?